test/schema-tests.js

```diff
@@ -13,10 +13,14 @@ describe('convict schema file', function() {
     }
   });
 
-  it('must parse a config specification from a file', function() {
+  beforeEach(function() {
```

Comment:
The code in that existing test was setting up `conf`, which actually needed to run before the rest of the tests could pass, so I changed it to run within `beforeEach` rather than relying on a test to run initially for some setup.

The new test I wrote for built-ins in schema files was affecting the rest of the tests by changing the `conf`. So I made sure that it was done specifically for my test.